src/Components/Greetings.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+function Greetings(props) {
+const {lang, children} = props;
+
+    function chooseLanguage(lang) {
+        if(lang === 'en') {
+            return <p>Hello</p>
+        }
+        else if(lang === 'de') {
+            return <p>Hallo</p>
+        }
+        else if(lang === 'es') {
+            return <p>Hola</p>
+        }
+        else if(lang === 'fr') {
+            return <p>Bonjour</p>
+        }
+    }
+
+  return (
+    <div>
+        {chooseLanguage(lang)} {children}
```

Comment:
Ideally, if you want to return the text, you should use a proper tag, in this case, it could be just a `<p>`, but it is always good practices if you use it.